docs/core/deploying/preparing-libraries-for-trimming.md

```diff
@@ -22,11 +22,13 @@ If your app only uses parts of a library that are compatible with trimming, cons
 
 These instructions show how to enable and resolve static analysis warnings to prepare a library for trimming. Follow these steps if you are authoring a library and either want to proactively make your library trimmable, or have been contacted by app authors who encountered trim warnings from your library.
 
-Use the .NET 6 SDK for the best experience. It is possible to [show analysis warnings](trimming-options.md#analysis-warnings) in .NET 5, but not all framework assemblies were annotated correctly in .NET 5. Also, this will show detailed warnings for every library, including framework libraries. These instructions assume you are using the .NET 6 SDK.
+Ensure you are using the .NET 6 SDK for these steps. They will not work correctly in previous versions.
 
-### Incomplete Roslyn analyzer
+## Enable Roslyn analyzer
 
-During development, you may set `<EnableTrimAnalyzer>true</EnableTrimAnalyzer>` (in .NET 6+) in your library project to get a _limited_ set of warnings from the Roslyn analyzer. This analyzer is incomplete and should only be used as a convenience. It is important to follow the next steps to ensure that your library is compatible with trimming.
+During development, set `<EnableTrimAnalyzer>true</EnableTrimAnalyzer>` (in .NET 6+) in your library project. This will not have any effect on the output, but it will enable trim analysis during build via a Roslyn analyzer.
```

Comment:
Why only during development?
If I want my library to be trim compatible - I should set this property... as simple as that.